src/NSwag.CodeGeneration/Models/OperationModelBase.cs

```diff
@@ -289,6 +289,9 @@ public string Produces
         /// <summary>Gets the operation extension data.</summary>
         public IDictionary<string, object> ExtensionData => _operation.ExtensionData;
 
+        /// <summary>Gets the operation tags </summary>
+        public string Tags => string.Join(",", _operation.Tags);
```

Comment:
I think we should expose the string array, allows more custom scenarios in templates and you can still join within the liquid templates if needed.

Reply:
Addressed this PR comment in pr #4807
Kindly review that PR